js/stores/appStore.js

```diff
@@ -633,55 +633,38 @@ const handleAppAction = (action) => {
     case AppConstants.APP_ADD_AUTOFILL_ADDRESS:
       {
         const Filtering = require('../../app/filtering')
-        appState = appState.setIn(['autofill', 'addresses', 'guid'],
-          appState.getIn(['autofill', 'addresses', 'guid']).filterNot((guid) => {
-            return guid === action.originalDetail.get('guid')
-          }))
-
-        let guids = appState.getIn(['autofill', 'addresses', 'guid'])
-        const guid = Filtering.addAutofillAddress(action.detail.toJS(),
+        Filtering.addAutofillAddress(action.detail.toJS(),
```

Comment:
don't worry about this right now, but we should pull these methods out of filtering and move them to the new `autofill.js` module you created at some point. They don't really need `Filtering` anymore  because they don't have to be sent to all the sessions